Makefile

```diff
@@ -284,25 +315,18 @@ endif
 ifeq ($(eligibility),)
 	echo "parameter <eligibility> missing" >&2 && exit 1
 endif
-	TS_NODE_PROJECT=./tsconfig.hardhat.json \
-	HOPR_ENVIRONMENT_ID="$(environment)" \
-	  yarn workspace @hoprnet/hopr-ethereum run hardhat register \
-   --network $(network) \
-   --task force-eligibility-update \
-   --native-addresses "$(native_addresses)"\
-   --eligibility "$(eligibility)"
+	make -C packages/ethereum/contracts force-eligibility-update \
```

Comment:
Please intend the 2 lines properly one more step. Same for the other occurrences below.